atlp-rwanda/ecommerce-app-predators-bn
# #36 Feature: update profile

####What does this PR do?

Updating User Profile endpoint   

######Description of Task?

Having the folowing endpoint working fine
GET /  api/profiles
GET / api/profiles/:id
PATCH /api/profile/:id

How should this be manually tested?
make sure you have github acount,
clone this repository the go to this branch
run npm run dev for starting server
go to postman user URL http://localhost:3000/api/profiles/:id (replace :id with number of id)
In Authorization choose ( bearer token) and put your token,
NB: the token must belong to the Id you are using in order to update your profile. if not you will get error message which says "you are not authorized to update the profile"


Any background context you want to provide?

To ensure security and prevent unauthorized access,  you should add your bearer token in authorization and that token should belong to the id you want to update

What are the relevant pivotal tracker/Trello stories?
https://trello.com/c/BMxojmxd/36-update-profile
`
Screenshots (if appropriate)
N/A

Questions:
N/A
